src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -372,6 +375,13 @@ export class StudyViewPageStore {
                 return acc;
             }, [] as CopyNumberGeneFilter[]);
         }
+        this.defaultFilters = filters;
+    }
+
+    @computed
+    get isInitialLoading(): boolean {
+        // TODO: figure out why isEqual cannot be applied to these two observables
+        return _.isEqual(JSON.parse(JSON.stringify(this.defaultFilters)), JSON.parse(JSON.stringify(this.filters)));
```

Comment:
it's probably because stuff mobx adds to make arrays and objects observable.   look at mobx "toJS"  https://mobx.js.org/refguide/tojson.html
will accomplish the same as what you are doing above but in more sane fashion

Reply:
I tried toJS yesterday. It's surprisingly not working here